template/Cake.Frosting.Template.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageType>Template</PackageType>
+    <PackageId>Cake.Frosting.Template</PackageId>
+    <Title>Cake.Frosting templates for the .NET SDK.</Title>
+    <Description>Cake.Frosting templates for the .NET SDK.</Description>
+    <Authors>Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman, Martin Björkström and contributors</Authors>
```

Comment:
I think we are missing some names here...

Reply:
Ah just copied
https://github.com/cake-build/frosting/blob/3f3c3cf4c849ff11d92d3374ea1bdbfe1ebf6287/src/Cake.Frosting/Cake.Frosting.csproj#L6

I'll create a Shared msbuild like we've got with Cake so all common properties in one place.

Reply:
That seems like a great idea!

Reply:
Moved to Shared.msbuild 
![image](https://user-images.githubusercontent.com/1647294/86858779-9ccb1f00-c0c1-11ea-8ec8-ce5a2f349339.png)